docs/ops/sequence/AUGRUSequence_dev.md

```diff
@@ -0,0 +1,168 @@
+# AUGRUSequence  {#openvino_docs_ops_sequence_AUGRUSequence_dev}
+
+**Versioned name**: *AUGRUSequence-dev*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUSequence* operation represents a series of AUGRU cells (GRU with attentional update gate).
+
+**Detailed description**: The main difference between *AUGRUSequence* and [GRUSequence](./GRUSequence_5.md) is the additional attention score input `A`, which is a multiplier for the update gate.
```

Comment:
Could you please describe idea of usage of `AUGRUSequence` shortly? I think it makes sense to add scientific explanation here and after you can refer to the original paper of it.

Reply:
I would like to finalize the discussion about the interface, formula, shapes and attributes of the operation.
I suggest to add proposals for more detailed description as a contribution in a sepatare PR.

Reply:
Agree, let us do it so.